src/Html2Text.php

```diff
@@ -10,30 +10,26 @@
 use DOMNode;
 use DOMProcessingInstruction;
 use DOMText;
-use Symfony\Polyfill\Mbstring\Mbstring;
 
 class Html2Text
 {
     public static function convert(string $html, Config $config = new Config()): string
     {
-        $isOfficeDocument = static::isOfficeDocument($html);
+        $isOfficeDocument = self::isOfficeDocument($html);
 
         if ($isOfficeDocument) {
             // remove office namespace
-            $html = str_replace(["<o:p>", "</o:p>"], '', $html);
+            $html = str_replace(['<o:p>', '</o:p>'], '', $html);
         }
 
-        $html = static::fixNewlines($html);
-        if (mb_detect_encoding($html, 'UTF-8', true)) {
-            $html = Mbstring::mb_convert_encoding($html, 'HTML-ENTITIES', 'UTF-8');
-        }
```

Comment:
This is about the only relevant change in this PR, it removes the use of the deprecated `mb_convert_encoding` method and resorts to handling conversions with `DOMDocument::loadHTML()`s built-in encoding discovery.